$NetBSD: patch-ac,v 1.12 2015/12/29 23:34:49 dholland Exp $

Fix build with recent kaffe changes:
- some header files are not used and don't even exist

--- libraries/clib/sound/esd/org_tritonus_lowlevel_esd_EsdStream.c.orig	2006-05-16 15:29:48.000000000 +0000
+++ libraries/clib/sound/esd/org_tritonus_lowlevel_esd_EsdStream.c
@@ -27,7 +27,7 @@
 #include	<stdio.h>
 #include	<unistd.h>
 #include	"common.h"
-#include	"org_tritonus_lowlevel_esd_EsdStream.h"
+/* #include	"org_tritonus_lowlevel_esd_EsdStream.h" */
 
 // TODO: remove, replace by debug_flag
 //static int	debug_flag = 0;
